atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -66,13 +68,9 @@ def wait_to_finish(self):
         self.build = self.osbs.wait_for_build_to_finish(self.name)
         return self.build
 
-    def watch_logs(self, logger):
+    def watch_logs(self):
         for line in self.osbs.get_build_logs(self.name, follow=True):
-            # TODO: This is a little clunky:
-            # 2017-02-24 14:22:51,472 - atomic_reactor.plugins.orchestrate_build - INFO -
-            # x86_64 - 2017-02-24 14:22:51,314 - atomic_reactor.plugin - INFO -
-            # <worker build log message>
-            logger.info('%s - %s', self.platform, line)
+            self.log.info("%s", line)
```

Comment:
So actually this was fine the way you had it, sorry I asked you to change it. :-/